plugins/whatsapp.js

```
@@ -4,367 +4,410 @@ import { convertNormalMessageToViewOnce, ModifyViewOnceMessage, toJid } from '#u
-			quoted: msg.quoted
-		});
-	}
+  {
+    pattern: 'quoted',
+    public: false,
+    type: 'whatsapp',
+    desc: 'quoted message',
+  },
+  async (message) => {
+    if (!message.reply_message) return await message.send('```Reply A Message```');
+    let key = message.reply_message.key.id;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'key' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)